pkg/common/common.go

```diff
@@ -109,4 +107,7 @@ const (
 
 	// ScratchSpaceNeededExitCode is the exit code that indicates the importer pod requires scratch space to function properly.
 	ScratchSpaceNeededExitCode = 42
+
+	// VolumeMode is environmet variable indicating the volume mode of the PV (block/FS)
```

Comment:
Just like with the importer, we don't have to pass the volume mode to the pod because the pod can determine by checking the existence of the block device. So we don't have to set the environment variable at all.